oauthenticator/globus.py

```diff
@@ -96,16 +113,6 @@ def _scope_default(self):
             'urn:globus:auth:scope:transfer.api.globus.org:all',
         ]
 
-    allow_refresh_tokens = Bool(
-        help="""Allow users to have Refresh Tokens. If Refresh Tokens are not
-        allowed, users must use regular Access Tokens which will expire after
-        a set time. Set to False for increased security, True for increased
-        convenience."""
-    ).tag(config=True)
-
-    def _allow_refresh_tokens_default(self):
-        return True
-
```

Comment:
Removing this since it turned out to be broken anyways. The arg to allow refresh tokens needs to be passed in on the first call to auth. I'm pretty sure someone broke this when adding 'authorize_url'. 

The new way of doing this is to pass in a dict to `extra_authorize_params`:

    c.LocalGlobusOAuthenticator.extra_authorize_params = {'access_type': 'offline', 'session_required_identities': 'a65e5bc6-ab8c-469a-9d1d-b33608da9ffb'}

Not sure if setting `'access_type': 'offline'` is too obscure, we could bring back the above `allow_refresh_tokens` option if we overrode their oauth2 class to inject the correct value.